src/managed/Microsoft.DotNet.PlatformAbstractions/Native/PlatformApis.cs

```diff
@@ -85,6 +85,8 @@ private static string GetDistroVersionId()
 
         private static DistroInfo LoadDistroInfo()
         {
+            DistroInfo result = null;
```

Comment:
FYI, we will have to service PlatformAbstractions and potentially DependencyModel for this change.

/cc @weshaggard 

See https://github.com/dotnet/core-setup/pull/3134/commits/0958309a443bf7f0423d788d2e69b922c25adc7f for an example of the changes necessary to service DependencyModel in `2.0.1`.

Reply:
@janvorli - let me know if you want me to tackle those changes.  I can push a commit to your PR.

Reply:
@eerhardt it would be awesome if you could do that.

Reply:
Ok.  I'm getting a repro on the JIT issue that is blocking the CLI uptake of the latest 2.1 runtime now.  I'll get this done later today.

Reply:
@eerhardt thank you!

Reply:
Pushed.  @weshaggard - can you take a look at the servicing changes?